modules/distribution/product/src/main/resources/conf/infer.json

```diff
@@ -76,9 +74,29 @@
       "database.apim_db.url": "jdbc:mysql://$ref{database.apim_db.hostname}:$ref{database.apim_db.port}/$ref{database.apim_db.name}",
       "database.apim_db.validationQuery": "SELECT 1"
     },
+    "oracle": {
+      "database.apim_db.driver": "oracle.jdbc.OracleDriver",
+      "database.apim_db.url": "jdbc:oracle:thin:@$ref{database.apim_db.hostname}:$ref{database.apim_db.port}/$ref{database.apim_db.sid}",
+      "database.apim_db.validationQuery": "SELECT 1 FROM DUAL"
+    },
+    "mssql": {
+      "database.apim_db.driver": "com.microsoft.sqlserver.jdbc.SQLServerDriver",
+      "database.apim_db.url": "jdbc:sqlserver://$ref{database.apim_db.hostname}:$ref{database.apim_db.port};databaseName\u003d$ref{database.apim_db.name};SendStringParametersAsUnicode\u003dfalse",
+      "database.apim_db.validationQuery": "SELECT 1"
+    },
+    "db2": {
+      "database.apim_db.driver": "com.ibm.db2.jcc.DB2Driver",
+      "database.apim_db.url": "jdbc:db2://$ref{database.apim_db.hostname}:$ref{database.apim_db.port}/$ref{database.apim_db.name}",
+      "database.apim_db.validationQuery": "SELECT 1 FROM sysibm.sysdummy1"
+    },
+    "postgre": {
+      "database.apim_db.driver": "org.postgresql.Driver",
+      "database.apim_db.url": "jdbc:postgresql://$ref{database.apim_db.hostname}:$ref{database.apim_db.port}/$ref{database.apim_db.name}",
+      "database.apim_db.validationQuery": "SELECT 1; COMMIT"
+    },
     "h2": {
       "database.apim_db.driver": "org.h2.Driver",
-      "database.apim_db.url": "jdbc:h2:./repository/database/WSO2AM_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000",
+      "database.apim_db.url": "jdbc:h2:./repository/database/WSO2CARBON_DB;DB_CLOSE_ON_EXIT\u003dFALSE;LOCK_TIMEOUT\u003d60000",
```

Comment:
What's the reason for this change?

Reply:
it should be support mysql,h2,mssql,postgre,oracle,db2 as mentioned in https://apim.docs.wso2.com/en/latest/Reference/ConfigCatalog/.

Reply:
oh, I just copy from `database.$1.type`.

Reply:
I don't know where repo `database.$1.type` is come from. I just peek from running instance.

Reply:
my question is why have you replaced `WSO2AM_DB` with `WSO2CARBON_DB `?

Reply:
oh, sorry. it's a mistake.

Reply:
it should be WSO2AM_DB